src/lib/project.ts

```diff
@@ -29,11 +27,6 @@ export interface ProjectInfo {
   description: string;
   defaultBranch: string;
   delegates: string[];
-  patches: {
```

Comment:
Ok I think I would still keep this part of patches, since the httpd already returns the count of patches.
Like seen here https://seed.radicle.xyz/api/v1/projects/rad:z3gqcJUoA1n9HaHKufZs5FCSGazv5.

And also keep it as part of the `Project` class.

The rest looks great. Black hole development :sweat_smile:

Reply:
5a299aab19e0a268c3c98e19b8746fd0a3e44739